### DISCUSSION TITLE
Question - CLI

### DISCUSSION THREAD

**Discussion by dexter74** [Chia-Network/chia-blockchain#2927] [Q&A]
Hello,

I would like to know the parameters of the command line to launch the miner?

Is this related to the "chia.exe start all" parameter? 

```
@echo off

set Utilisateur=
set version=

:: Start Service :
C:\Users\%Utilisateur%\AppData\Local\chia-blockchain\app-%version%\resources\app.asar.unpacked\daemon/chia.exe run_daemon  
C:\Users\%Utilisateur%\AppData\Local\chia-blockchain\app-%version%\resources\app.asar.unpacked\daemon/chia.exe start node
C:\Users\%Utilisateur%\AppData\Local\chia-blockchain\app-%version%\resources\app.asar.unpacked\daemon/chia.exe start harvester
C:\Users\%Utilisateur%\AppData\Local\chia-blockchain\app-%version%\resources\app.asar.unpacked\daemon/chia.exe start farmer
C:\Users\%Utilisateur%\AppData\Local\chia-blockchain\app-%version%\resources\app.asar.unpacked\daemon/chia.exe start farmer-only
C:\Users\%Utilisateur%\AppData\Local\chia-blockchain\app-%version%\resources\app.asar.unpacked\daemon/chia.exe start wallet
```

**Comment by cccat6** [comment]
See here https://github.com/Chia-Network/chia-blockchain/wiki/CLI-Commands-Reference
Like this example `chia plots create -k 32 -b 4000 -t /path/to/temporary/directory -d /path/to/final/directory`
By the way, did you use the slash wrong? "/" or "\\"?

**Comment by dexter74** [comment]
Hello,

What I am asking is not the creation of the plots but the launch of the miner when the plots are created.

**Comment by Last-Airbender** [comment]
It's commonly used for a plotter + farming machine in one unit. You can stop each of them individually in CLI if you have different machines for plotting only and farming only.

**Comment by dexter74** [comment]
Hello,
I would like to run the miner as a task or service from the command line.

**Reply by dorkmo** [reply]
you can do `chia start node farmer` to get the basic system running

**Comment by dexter74** [comment]
Hello,
Does the background process initiate synchronization of the node as the client at the blockchain level ?
